Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -3355,12 +3355,12 @@ def run_polling_command(client: Client, cmd: str, args: Dict[str, Any]):
         return final_command_results
     else:
         scheduled_command = ScheduledCommand(command=cmd, next_run_in_seconds=interval, args=args, timeout_in_seconds=timeout)
-        command_results = CommandResults(scheduled_command=scheduled_command)
-        return command_results
+        return_results(CommandResults(scheduled_command=scheduled_command))
```

Comment:
```suggestion
        return CommandResults(scheduled_command=scheduled_command)
```
You're already calling the return_results() in the main function for this command